libs/ui-lib/lib/common/config/constants.ts

```diff
@@ -27,7 +27,7 @@ export const getOpenShiftNetworkingDocsLink = (ocpVersion?: string) =>
 export const CLUSTER_MANAGER_SITE_LINK = 'https://console.redhat.com/openshift/install/pull-secret';
 export const PULL_SECRET_INFO_LINK = CLUSTER_MANAGER_SITE_LINK;
 export const ODF_REQUIREMENTS_LINK =
-  'https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2022/html/assisted_installer_for_openshift_container_platform/assembly_installing-operators#proc_installing-openshift-data-foundation_assembly_installing-operators';
+  'https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.15';
```

Comment:
Again, hard-coded version numbers are problematic. Better to leave "4.15" out of the URL so that it resolves to the latest version.